admin_ui/src/components/FormAdd.vue

```diff
@@ -6,12 +6,30 @@
             <p v-if="formConfig.description">{{ formConfig.description }}</p>
         </template>
 
-        <pre>{{ errors }}</pre>
+        <div v-show="successMessage">
+            <p>{{ successMessage }}</p>
+            <ul>
+                <li>
+                    <a href="#" @click.prevent="resetForm">Use again</a>
+                </li>
+                <li>
+                    <router-link to="/">Back to home page</router-link>
+                </li>
+            </ul>
+        </div>
```

Comment:
I would show only a success message  to emphasize this a little (not form name and description). Something like this
```Vue
<template v-if="formConfig && !successMessage">
    <h1>{{ formConfig.name }}</h1>

    <p v-if="formConfig.description">{{ formConfig.description }}</p>
</template>

<div v-show="successMessage">
    <p id="success">{{ successMessage }}</p> <-- let it be bold and @green to be highlighted or something
    <ul>
        <li>
            <a href="#" @click.prevent="resetForm">Use again</a>
        </li>
        <li>
            <router-link to="/">Back to home page</router-link>
        </li>
    </ul>
</div>
```
We also need to add something like this to change the style of the success message
```less
<style scoped lang="less">
@import "../vars.less";

h1 {
    text-transform: capitalize;
}
#success {
    color: @green;
    font-weight: bold;
    font-size: 1.2rem;
}
</style>
```
![success_message](https://user-images.githubusercontent.com/30960668/176351056-6bc59fed-10f2-4889-a704-d50549d104ac.png)

Reply:
It's a good idea - the page is a bit busy at the moment.

We can't emphasis the response message too much, in case someone returns something really long (for example, some JSON).